extmod/modframebuf.c

```diff
@@ -240,11 +240,11 @@ STATIC mp_framebuf_p_t formats[] = {
     [FRAMEBUF_MHMSB] = {mono_horiz_setpixel, mono_horiz_getpixel, mono_horiz_fill_rect},
 };
 
-static inline void setpixel(const mp_obj_framebuf_t *fb, int x, int y, uint32_t col) {
+static inline void setpixel(const mp_obj_framebuf_t *fb, unsigned int x, unsigned int y, uint32_t col) {
```

Comment:
Instead of `int` --> `unsigned int` can we use `mp_uint_t`, and then change the various `mp_int_t x = mp_obj_get_int(args[1]);` lines to `mp_uint_t x = mp_obj_get_int(args[1]);`

Reply:
Yeah, I was wondering what the best thing was to do here.  Do you first of all agree that an unsigned type is the right choice, since already clipping is done before it gets to these functions?

The problem with `mp_uint_t` is that it's large on nan-boxing ports (ie 64-bit on a 32-bit arch) so less efficient.  And `unsigned int`'s range should suffice.

Reply:
This is left as `unsigned int`.